src/comp_loinc/main.py

```diff
@@ -111,31 +111,36 @@ def get_loinc_version():
 
 
 @app.callback()
-def comp_loinc_main(loinc_dir: Annotated[pathlib.Path, typer.Option()] = get_latest_loinc_release(),
-                    trees_date: Annotated[str, typer.Option()] = get_latest_trees(),
-                    loinc_version: Annotated[str, typer.Option()] = get_loinc_version(),
-                    out_dir: Annotated[pathlib.Path, typer.Option()] = PROJECT_DIR / 'data' / 'loinc_owl',
-                    log_level: Annotated[str, typer.Option()] = 'WARN',
-                    owl_output: Annotated[bool, typer.Option()] = True,
-                    rdf_output: Annotated[bool, typer.Option()] = True
-                    ):
-    logging.basicConfig(filename=PROJECT_DIR / 'logs' / f'log_{time.strftime("%Y%m%d-%H%M%S")}.txt',
-                        encoding='utf-8',
-                        level=logging.getLevelName(log_level.upper())
-                        )
+def comp_loinc_main(
+    loinc_dir: Annotated[pathlib.Path, typer.Option()] = get_latest_loinc_release(),
+    trees_date: Annotated[str, typer.Option()] = get_latest_trees(),
+    loinc_version: Annotated[str, typer.Option()] = get_loinc_version(),
+    out_dir: Annotated[pathlib.Path, typer.Option()] = PROJECT_DIR / 'data' / 'loinc_owl',
+    log_level: Annotated[str, typer.Option()] = 'WARN',
+    owl_output: Annotated[bool, typer.Option()] = True,
+    rdf_output: Annotated[bool, typer.Option()] = True
+):
+    """Runs before any command."""
+    logging.basicConfig(
+        filename=PROJECT_DIR / 'logs' / f'log_{time.strftime("%Y%m%d-%H%M%S")}.txt',
+        encoding='utf-8',
+        level=logging.getLevelName(log_level.upper())
+    )
 
     global release, generator
     release = loinclib.Graph(loinc_dir, loinc_version, loinc_dir / 'trees' / trees_date)
-    generator = comp_loinc.Generator(loinc_release=release,
-                                     schema_directory=pathlib.Path(SCHEMA_DIR),
-                                     output_directory=out_dir,
-                                     owl_output=owl_output,
-                                     rdf_output=rdf_output
-                                     )
+    generator = comp_loinc.Generator(
+        loinc_release=release,
+        schema_directory=pathlib.Path(SCHEMA_DIR),
+        output_directory=out_dir,
+        owl_output=owl_output,
+        rdf_output=rdf_output
+    )
 
 
 @app.command()
 def generate_all():
+    """Generate the entire ontology, utilizing the primary parts model, but not the supplementary."""
```

Comment:
## Added docstrings / code commenting
I think this is correct (though it has supplementary defs). Maybe this could be phrased better.